### DISCUSSION TITLE
为什么没有成果页？

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#294] [Q&A]
您好，
我看了下新版本，貌似没有成果页。为什么呢？
谢谢！

**Comment by SwitWu** [comment]
自己写一个 chapter 不就行吗

**Reply by earthmoon** [reply]
您好，

自己写的有序号，如果去掉序号又不会进目录，居中间隔也有难度。如果你们后面方便，可以帮忙添加这个。

谢谢！
祝一切如意。

**Reply by SwitWu** [reply]
你看看 cls 里面的致谢是如何定义的，应该大差不差

**Reply by earthmoon** [reply]
您好，

原来如此，我去试试。

谢谢！
祝一切如意。
